src/VisualStudio/Core/Def/Implementation/TodoComments/VisualStudioTodoCommentsService.cs

```diff
@@ -92,8 +95,10 @@ private async Task StartAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task StartWorkerAsync(CancellationToken cancellationToken)
+        private async Task StartWorkerAsync()
         {
+            var cancellationToken = ThreadingContext.DisposalToken;
```

Comment:
Consider having StartWorkerAsync still take the cancellation token and then passing this in StartAsync(); I otherwise was scratching my head why StartAsync() was catching OperationCancelled exceptions when it wasn't passing a token in.